src/heronarts/lx/parameter/CompoundParameter.java

```diff
@@ -129,7 +129,7 @@ public float getBaseValuef() {
   }
 
   public double getBaseNormalized() {
-    return super.getNormalized(getBaseValue());
+    return super.range.getNormalized(getBaseValue());
```

Comment:
This should stay super.getNormalized()... in case the super method ever changes.

Reply:
Great.